ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -485,6 +485,45 @@ interface TestController {
      * @param hooks - The set of RequestHook subclasses.
      */
     removeRequestHooks(...hooks: object[]): TestControllerPromise;
+    /**
+     * Returns cookies associated with the specified cookie objects. If there are no parameters, this method returns all cookies.
+     *
+     * @param cookies - The cookie objects.
+     */
+    getCookies(cookies?: CookieOptions | CookieOptions[]): Promise<CookieOptions[]>;
+    /**
+     * Returns cookies with the specified names and URL contexts.
+     *
+     * @param names - The cookie names.
+     * @param urls - The URL.
+     */
+    getCookies(names: string | string[], urls?: string | string[]): Promise<CookieOptions[]>;
+    /**
+     * Sets the specified cookie objects.
+     *
+     * @param cookies - The cookie objects.
+     */
+    setCookies(cookies?: CookieOptions | CookieOptions[]): TestControllerPromise;
+    /**
+     * Sets cookies' name-value pairs to the specified URL contexts.
```

Comment:
```suggestion
     * Matches cookie name-value pairs to the specified URLs.
```

how many URLs can this method accept? Is it just one? if so there's no need to pluralise...

```suggestion
     * Assigns the specified URL to the specified cookie name-value pairs.
```

Reply:
only one